src/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -141,58 +126,92 @@ public static void InvalidConstructors()
             }
         }
 
-        [Fact]
-        public static async Task InvalidDates()
+        [Theory]
+        [InlineData("CDoffsetOutOfBounds.zip")]
+        [InlineData("EOCDmissing.zip")]
+        public static async Task ZipArchive_InvalidStream(String zipname)
         {
-            using (ZipArchive archive = new ZipArchive(await StreamHelpers.CreateTempCopyStream(
-                 ZipTest.bad("invaliddate.zip")), ZipArchiveMode.Read))
-            {
-                Assert.Equal(new DateTime(1980, 1, 1, 0, 0, 0), archive.Entries[0].LastWriteTime.DateTime); //"Date isn't correct on invalid date"
-            }
-
-            using (ZipArchive archive = new ZipArchive(new MemoryStream(), ZipArchiveMode.Create))
-            {
-                ZipArchiveEntry entry = archive.CreateEntry("test");
-                Assert.Throws<ArgumentOutOfRangeException>(() =>
-                { entry.LastWriteTime = new DateTimeOffset(1979, 12, 3, 5, 6, 2, new TimeSpan()); }); //"should throw on bad date"
-                Assert.Throws<ArgumentOutOfRangeException>(() =>
-                { entry.LastWriteTime = new DateTimeOffset(2980, 12, 3, 5, 6, 2, new TimeSpan()); }); //"Should throw on bad date"
-            }
+            string filename = ZipTest.bad(zipname);
+            using (var stream = await StreamHelpers.CreateTempCopyStream(filename))
+                Assert.Throws<InvalidDataException>(() => new ZipArchive(stream, ZipArchiveMode.Read));
         }
 
-        [Fact]
-        public static async Task StrangeFiles1()
+        [Theory]
+        [InlineData("CDoffsetInBoundsWrong.zip")]
+        [InlineData("numberOfEntriesDifferent.zip")]
+        public static async Task ZipArchive_InvalidEntryTable(String zipname)
         {
-            ZipTest.IsZipSameAsDir(await StreamHelpers.CreateTempCopyStream(
-                 ZipTest.strange(Path.Combine("extradata", "extraDataLHandCDentryAndArchiveComments.zip"))), ZipTest.zfolder("verysmall"), ZipArchiveMode.Update, false, false);
+            // using (ZipArchive archive = ZipFile.OpenRead(filename))
```

Comment:
It's a relic of the old tests that I neglected to remove. The same kind of comment appears a few times throughout the file.